core/templates/services/playthrough.service.ts

```diff
@@ -40,12 +40,14 @@ class CyclicStateTransitionsTracker {
   /** A path of visited states without any repeats. */
   private pathOfVisitedStates: string[];
   /** The most recently discovered cycle of visited states. */
-  private cycleOfVisitedStates: string[];
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
```

Comment:
Done

Reply:
Could you please use two sentences and the proper punctuation (comma after "information") as described in my previous comment? Ditto elsewhere.

Otherwise you end up with what is called a ["run-on sentence"](https://lsa.umich.edu/sweetland/undergraduates/writing-guides/what-is-a-run-on-sentence---how-do-i-fix-it-.html).

Reply:
Yes Correct, but we are using this type of comment in whole codebase. We have around 150+ such same exact comments. Is it okay to change this file comment only??